### PR TITLE
Add speaker's corner page

### PR DESCRIPTION
**Pull request by everthemore**

### CODE REVIEW

File: speakerscorner.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Speaker's Corner
+
+Named after the renowned corner of Hyde Park in London, the Speaker's Corner seminars are meant to provide a platform
+for everyone who would like to share their research.
+
+## Give a talk
+The series is hence on a "self-invitation" basis, where you register for giving a talk through the [Registration Form](#speakerscornerregistration) and
```
Comment (akhmerov) [inline speakerscorner.md:7]:
```suggestion
The series is hence on a "self-invitation" basis, where you register for giving a talk through the [Registration Form](https://github.com/virtualscienceforum/virtualscienceforum/issues/new?template=speakers_corner_application.md) and
```

The form is not online yet, but that's its future URL

**Comment by akhmerov** [comment]
We should link back from the mailing list registration form (there's a link that resolves to nothing right now)

File: speakerscorner.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Speaker's Corner
+
+Named after the renowned corner of Hyde Park in London, the Speaker's Corner seminars are meant to provide a platform
```
Comment (akhmerov) [inline speakerscorner.md:3]:
```suggestion
Named after the renowned corner of Hyde Park in London, the Speaker's Corner seminars are a platform
```

Comment (everthemore) [reply speakerscorner.md:3]: Thanks for the updates. This was meant as a quick draft just to get a page started. But I'm happy to start tweaking from here regardless 👍

**Comment by everthemore** [comment]
> We should link back from the mailing list registration form (there's a link that resolves to nothing right now)

done

**Comment by everthemore** [comment]
> > We should link back from the mailing list registration form (there's a link that resolves to nothing right now)
> 
> done

or perhaps not; does the HTML href resolve # as a local anchor?

**Comment by akhmerov** [comment]
Following the org meeting review #154, merging.

**Pull request merged**